apps/angular/simple-animations/src/app/app.component.ts

```diff
@@ -17,9 +25,32 @@ import { Component } from '@angular/core';
       }
     }
   `,
+  animations: [
+    trigger('slideIn', [
+      transition(':enter', [
+        style({ transform: 'translateX(-100%)' }),
+        animate('1000ms ease-in', style({ transform: 'translateX(0%)' })),
+      ]),
+    ]),
+    trigger('stagger', [
+      transition(':enter', [
+        query('.list-item', [
+          style({ opacity: 0, transform: 'translateX(-100px)' }),
+          stagger(500, [
+            // for event loop -> better to use 250 increments (250,500,750,1000)?
+            animate(
+              '500ms cubic-bezier(0.35, 0, 0.25, 1)',
+              style({ opacity: 1, transform: 'none' }),
+            ),
+          ]),
+        ]),
+      ]),
+    ]),
+  ],
   template: `
     <div class="mx-20 my-40 flex gap-5">
-      <section>
+      <!-- The posts could be inside *ngFor loop but for an animation to work a trackBy function is required  -->
```

Comment:
That's interesting, nice hint 👍 i'll try it with a `@for` loop to see if it works.